src/includes/constants.php

```diff
@@ -197,7 +197,15 @@
 
 // Define the temporary folder if it wasn't defined yet
 if (!defined('TEMP_PATH')) {
-    if (defined('TEMP_FOLDER')) {
+    // Check if path has been set in environment variables
+    if ($path = Environment::getEnv('SS_TEMP_PATH')) {
+        // If path is relative, rewrite it to be relative to BASE_PATH - as web requests are relative to
+        // public/index.php, and we don't want the TEMP_PATH to be inside the public/ directory by default
+        if (ltrim($path, DIRECTORY_SEPARATOR) === $path) {
+            $path = BASE_PATH . DIRECTORY_SEPARATOR . $path;
+        }
+        define('TEMP_PATH', $path);
+    } elseif (defined('TEMP_FOLDER')) {
```

Comment:
TEMP_FOLDER should still have higher precedence of SS_TEMP_PATH since it's an old alias for TEMP_PATH?

Seems slightly less likely to break existing setups if TEMP_FOLDER is read first

Reply:
I’ve swapped these around so `TEMP_FOLDER` takes priority as before